app/src/pages/inside/dashboardItemPage/modals/common/widgetControls/cumulativeTrendControls.jsx

```diff
@@ -124,18 +162,35 @@ export class CumulativeTrendControls extends Component {
             onFormAppearanceChange={onFormAppearanceChange}
           />
         </FieldProvider>
+
+        <TogglerControl fieldLabel=" " items={tabItems} value={2} />
```

Comment:
Should it always has `value={2}`?
What is the purpose of using this control without `FieldProvider`?

Reply:
The toggler should be disabled for the time being, it is for demo purposes only (API returns only latest launches), that's why it has static value and has no fieldProvider